### PR TITLE
Update list of supported distros #27

### LINKED ISSUE #27: Update list of supported distros
The supported distributions seem to be outdated. We should review the list, add new ones (Debian 11, Ubuntu etc.) and maybe remove what's out of support (since sometimes users work with old hosts, this step is optional)

The change should be reflected in `meta/main.yml` and within the `.github` directory so we don't list something as suported that's not tested yet.

### PR DESCRIPTION
**Pull request by danopt**
```
  - name: Rocky Linux
    versions:
    - "8"
    - "9"
```
**That's in the meta/main.yml of my pull request. I think we should remove it because of the following circumstances:**

- Rocky Linux **not found** in Galaxy REST API. See [https://galaxy.ansible.com/api/v1/platforms/?search=rocky](https://galaxy.ansible.com/api/v1/platforms/?search=rocky
)
- The documentation ( src: [https://galaxy.ansible.com/docs/contributing/creating_role.html](https://galaxy.ansible.com/docs/contributing/creating_role.html) ) mentioned the following:
_"Required. Provide a list of **valid platforms, and for each platform, a list of valid versions**. The obvious question of course is, where does one find the list of valid platforms? You can find the [list of platforms here](https://galaxy.ansible.com/api/v1/platforms/). The list is paginated. Click on the next_link value to get to view the next page. It’s not the prettiest interface, but for now, it works."_

Of course, we should also remove it in .github/workflow/molecule.yml.

### CODE REVIEW

**Comment by widhalmt** [comment]
I just want to leave a short summary of what we talked offline about.

CentOS < 8 and Rocky Linux are combined with RHEL as `EL`. At least that's what I learned. CentOS > 7 has it's own code and could be listed extra. At the moment I don't know if there are separate containers available for testing with Rocky Linux 8 and CentOS 8 or CentOS Stream. Rocky has by far higher priority I'd say.

File: .github/workflows/molecule.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
-      # max-parallel: 4
+      max-parallel: 4
       matrix:
-        distro: [centos7, debian10, rockylinux8]
+        distro: [centos7, centos8, debian10, debian11, rockylinux8, rockylinux9, ubuntu2004, ubuntu2204]
```
Comment (widhalmt) [inline .github/workflows/molecule.yml]: Looks like `rockylinux9` changed something with the way `dnf` handles repository-keys. Maybe we need to have a testing host to try this step "manually".

Comment (danopt) [reply .github/workflows/molecule.yml]: Workaround has been implented #30

File: meta/main.yml [hunk 2]
```diff
@@ -16,6 +16,15 @@ galaxy_info:
...
     - buster
+    - bullseye
+  - name: Ubuntu
+    versions:
+    - focal
+    - jammy
+  - name: Rocky Linux
+    versions:
+    - "8"
+    - "9"
```
Comment (widhalmt) [inline meta/main.yml]:
```suggestion
```
As far as I know, Rocky is still not in the list of supported OS'es, right? It's part of `EL`, IIRC.

File: .github/workflows/molecule.yml (hunk 1 above)
Comment (widhalmt) [reply .github/workflows/molecule.yml]: Thanks!

File: meta/main.yml (hunk 2 above)
Comment (widhalmt) [reply meta/main.yml]: Sorry, linter seems to be ok with it. So I'll close this. Sorry for the spam. :-)

Comment (widhalmt) [reply meta/main.yml]: Well, now - I was too fast again. It's still listed as lint.

**Comment by danopt** [comment]
As discussed with @widhalmt we will remove Ubuntu 20.04 and Ubuntu 22.04 from molecule tests. I will open a new Issue to propose support for geerlingguy's Docker Images _[docker-ubuntu2004-ansible](https://github.com/geerlingguy/docker-ubuntu2004-ansible)_ and _[docker-ubuntu2204-ansible](https://github.com/geerlingguy/docker-ubuntu2204-ansible)_ in the molecul test.

**Review by widhalmt — APPROVED** [review]
Nice. That should do the trick. Thank you!

**Pull request merged**